cityofaustin/atd-moped
# Adds line breaks and auto-sizing to various columns on the project list

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9853

This adjusts a number of columns on the project list so that they (1) have linebreaks between list items and (2) grow to the width of their content:

- project name
- team
- task orders
- contracts
- contract numbers
- tags
- partners
- funding

After user's react to this change, we may want to start truncating long strings in the cell render (task orders, team members with lots of roles....). My opinion is that if we're going to have lists of things in table cells, those things should never wrap. I did reach out to AMD and PDD for feedback on task orders specifically.

I will need to resolve conflicts once https://github.com/cityofaustin/atd-moped/pull/857 is merged.

## Testing
**URL to test:** [Deploy preview](https://deploy-preview-859--atd-moped-main.netlify.app/)

**Steps to test:**
1. Add multiple team members with multiple roles to a project
2. Add multiple task orders to a project
3. Add multiple contracts to a project
4. Add multiple tags to a project
5. In the project list view, observe that each of these columns display line-separated values and do not wrap
6. Remove the task orders and team members from the project
7. In the project list view, observe that the column widths for these fields minimal

---
#### Ship list
